src/controller/CommissioningDelegate.h

```diff
@@ -550,7 +568,8 @@ class CommissioningParameters
     Optional<bool> mAttemptWiFiNetworkScan;
     Optional<bool> mAttemptThreadNetworkScan; // This automatically gets set to false when a ThreadOperationalDataset is set
     Optional<bool> mSkipCommissioningComplete;
-    bool mCheckForMatchingFabric = false;
+    ICDRegistrationStrategy mICDRegistrationStrategy = ICDRegistrationStrategy::kBeforeComplete;
```

Comment:
default value should be kIgnore? Unless application set the actual strategy, at default it should go with kIgnore. thanks

Reply:
done